services/nginx/config/default.conf

```
@@ -25,6 +25,12 @@ server {
         return 302 /v3$request_uri;
     }
 
+    # redirect /<category>/<curie> to /<curie>
+    location ~^/[a-z]+/([^:]:[^:])$ {
```

(Thanks for the quick fix on my last comment, and sorry for bringing the same line so soon after!)

I was thinking through this regex a bit and I'm a little concerned that it may be too permissive; it'd catch, e.g., `/<alphabet-letters>/<anything with a single colon in it, even multiple levels down>`. I don't know how many possible values of `<category>` there are, but perhaps if there aren't too many we could explicitly name them here?

Also, I assume you want to match more than a single non-colon on either side of the colon, so you probably would want to add a `+` to each. Finally, I'm not 100% sure, but I think you might need a space between the tilde and the initial caret. Here's how I'd modify it:

```suggestion
    location ~ ^/[a-z]+/([^/:]+:[^/:]+)$ {
```

This also prevents it from descending into multiple levels, but you can remove the slash from the first character group to get rid of that behavior. I'd prefer it even more if the categories were explicitly specified, and the patterns on either side of the colon could be limited to just the legal characters for those identifiers.

---

I was having the same thoughts about the super permissive catch for categories, I could enumerate the ones that would actually work well, and if somebody comes in with something like `/case/MONARCH:123` it might as well go to search anyway. 

I think I'll also move the redirect over to the router, it's so much easier to test that way.

---

I take it back, I think I'm sticking with the nginx redirect, I struggled for a bit with the Vue router syntax and decided that I wanted to finish it more than I wanted to track down how to do a 301 redirect from Vue.